### PR TITLE
add traslation and little fix on css

### PR DESCRIPTION
**Pull request by pjroggy**

### CODE REVIEW

File: client/src/services/DestinationCardService.jsx
Comment (FabienRollet) [inline client/src/services/DestinationCardService.jsx:1]: Why leaving this file empty ? better be deleted if no used

File: client/src/styles/DestinationCard.scss [hunk 1]
```diff
@@ -44,17 +45,17 @@
 }
 
 .modal-overlay {
-  background: #023b8683;
+  background: #552d895e;
```
Comment (FabienRollet) [inline client/src/styles/DestinationCard.scss]: You should use a variable

File: client/src/styles/DestinationCard.scss [hunk 2]
```diff
@@ -44,17 +45,17 @@
...
   border-radius: 16px;
   box-shadow: 0 4px 30px #0000001a;
```
Comment (FabienRollet) [inline client/src/styles/DestinationCard.scss]: same

File: client/src/styles/ErrorPage404.scss [hunk 3]
```diff
@@ -12,11 +12,14 @@
 }
 
 .error404 {
-  @each $property, $value in $glassmorphisme {
-    #{$property}: $value;
-  }
+  background: #552d895e;
+  border-radius: 16px;
+  box-shadow: 0 4px 30px #0000001a;
```
Comment (FabienRollet) [inline client/src/styles/ErrorPage404.scss]: same

File: client/src/components/ReservationModule.jsx [hunk 4]
```diff
@@ -46,7 +46,7 @@ export default function ReservationModule() {
       />
       <DropdownTraveller />
       <button className="reservationButton" type="submit">
-        Réserver
+        Booking
```
Comment (FabienRollet) [inline client/src/components/ReservationModule.jsx]: Booking -> Book now

**Review by FabienRollet — CHANGES_REQUESTED** [review]

File: client/src/services/DestinationCardService.jsx
Comment (pjroggy) [reply client/src/services/DestinationCardService.jsx:1]: it seems this page is already deleted

**Review by FabienRollet — APPROVED** [review]

**Review by lxndrMr — APPROVED** [review]

**Pull request merged**